README.md

```diff
@@ -250,15 +217,47 @@ const entries = await scopedPlainClient.entry.getMany({
 })
 ```
 
-The benefits of using the plain version of the library are:
+You can try and change the above example at [Tonic](https://tonicdev.com/npm/contentful-management).
```

Comment:
This redirects to https://npm.runkit.com/contentful-management - should we link this idrectly?